modules/experimental-layers/src/path-marker-layer/path-marker-layer.js

```diff
@@ -49,11 +49,11 @@ export default class PathMarkerLayer extends CompositeLayer {
     if (coordinateSystem === COORDINATE_SYSTEM.METER_OFFSETS) {
       const [dx, dy] = viewport.metersToLngLatDelta(xyz);
       const [x, y] = coordinateOrigin;
-      return viewport.projectFlat([x - dx, dy + y]);
+      return viewport.projectFlat([x + dx, dy + y]);
```

Comment:
Add a HACK/TODO comment?

Reply:
The previous code was the "hacky" one. This one is the proper one.

Reply:
Yes, when we fix the `project module`, this will remain the same.